practice/flutter/flutter/logic.py

```diff
@@ -3,13 +3,101 @@
 from . import models
 
 
-def create_and_save_new_flutt(text):
-    """Creates and saves a new flutt.
+def create_and_save_new_flutt(text, timestamp=''):
```

Comment:
Minor comment: Generally if you want an optional argument that doesn't have a value, use `None` as the default value.